controllers/userController.js

```diff
@@ -13,3 +14,16 @@ exports.registerUser = asyncHandler(async (req, res, next) => {
     .status(201)
     .send({ success: true, message: "User created successfully", createdUser });
 });
+
+
+exports.signIn = asyncHandler(async(req,res) => {
+  User.findOne({email: req.body.email}),
+  function(err,user) {
+    if(err) throw err;
+    if(!user || !user.comparePassword(req.body.password)) 
+    return next(new ErrorResponse("Authentication failure", 401)); 
+  }
+  return res.json({
+    //How do I return?
```

Comment:
You can return the token and the user object in the json body for the timebeing. Will add the token to the cookie in the future